handlers/bounty_test.go

```diff
@@ -1014,3 +1014,53 @@ func TestGetBountyIndexById(t *testing.T) {
 		assert.Len(t, responseData, 0)
 	})
 }
+
+func GetAllBounties(t *testing.T) {
+	mockDb := dbMocks.NewDatabase(t)
+	mockHttpClient := mocks.NewHttpClient(t)
+	bHandler := NewBountyHandler(mockHttpClient, mockDb)
```

Comment:
@gouravmpk you are  testing to get created bounties here instead of getting all bounties `gobounties/all` `GetAllBounties()`